### ISSUE TITLE
To do list into early next week 

### ISSUE THREAD

**Issue by bpbond** [R-gon18/suli-tempestx#7]
1. Delete now-unnecessary stuff (pressure plot, babynames, etc) from analysis script
Commit this change and push

2. Make a Git branch and switch to it (in GitHub Desktop)

3. Reorganize R Markdown for clear logical flow for the reader. Use R Markdown headings (the “#”, “##”, etc.) and ggplot graph titles to set up an outline for the results, with parts:

* Basic data summary (report # of observations, # of collars, date range, mean and max fluxes)
* Graphing raw fluxes over time (after screening out zero values), color by Treatment
* Box plots by treatment
* Statistics

4. Open a pull request to the main branch and ask @ahopple or me to review

Statistics details — how to test for treatment differences using ANOVA
If useful/needed, review “analysis of variance”
The basic function in R for fitting a linear model is “lm”. Take a look at examples and give them a try
Basically we want to start by asking “is the flux a function of treatment”, i.e. does the treatment grouping affect the measured flux significantly (very much like the weight ~ group example)
You are going to want code like “mod_co2 <- lm( <formula>, data = <your data>)”
Then you can do “summary(mod_co2)” and “anova(mod_co2)” for example

**Comment by bpbond** [comment]
Abstract!

* Find exact due date—I think very soon
* @R-gon18 to start a google doc and share with Anya and me please
* Talk about abstract structure—how to write one
* They want the abstract submitted to Information Release

Structure:

* 1-2 sentences of general background
* 1-2 sentences of specific science problem
* What was done
* Main finding(s)
* Implications and next steps
